azure-pipelines.yml

```diff
@@ -47,6 +47,7 @@ jobs:
     additional_repositories:
     - name: tk-framework-shotgunutils
     - name: tk-multi-publish2
+    - name: tk-framework-qtwidgets
     - name: tk-maya
     - name: tk-config-basic
       ref: v1.3.0
```

Comment:
Should we update tk-config-basic version? Why is there a fixed version?

Reply:
I'll add a comment in the file.